test/exchange/OrderbookTests.js

```diff
@@ -61,7 +61,7 @@ describe('Orderbook Contract tests', () => {
 
     it('Supports the Address resolver Interface', async () => {
       // IOrderbook Interface
-      expect(await orderbook.supportsInterface("0x0950d870")).to.equal(true);
+      expect(await orderbook.supportsInterface("0x2a24311c")).to.equal(true);
```

Comment:
Is this hash value the same as the "orderbook_hash" variable?

Reply:
No, this interface id is the hash for all function signatures in the IOrderbook interface. This changes if we update the contract interface.

The CONTRACT_HASH has to be static and non-changing because it's the hash id of the contract in the AddressRegistry (used by the frontend and internal contracts). We can't use the interface id here because it changes over time.